docker/docker-entrypoint.sh

```diff
@@ -60,11 +63,19 @@ else
   echo "import runpy; runpy.run_path('../startup_scripts')" | ./manage.py shell --interface python
 fi
 
-# Copy static files
-./manage.py collectstatic --no-input
-
 echo "✅ Initialisation is done."
 
+(
+  UNIT_SOCKET="/opt/unit/unit.sock"
+  UNIT_CONFIG="/etc/unit/nginx-unit.json"
+  while [ ! -S $UNIT_SOCKET ]; do echo "$0: Waiting for control socket to be created..."; sleep 1; done
+  # even when the control socket exists, it does not mean unit has finished initialisation
+  # this curl call will get a reply once unit is fully launched
+  curl -s -X GET --unix-socket $UNIT_SOCKET http://localhost/
```

Comment:
I can't find the reference anymore, but I've recently learned that using the long version of arguments is beneficial in shell scripts to reduce the mental load of people reading the shell script later on. In this case this would mean to expand the `-s` and `-X` arguments as follows:

```suggestion
  curl --silent --request GET --unix-socket $UNIT_SOCKET http://localhost/
```

Reply:
Sounds logical. Will change.